model_hub/examples/huggingface/question-answering/qa_trial.py

```diff
@@ -166,13 +166,13 @@ def build_validation_data_loader(self) -> det_torch.DataLoader:
     def evaluate_batch(self, batch: det_torch.TorchData, batch_idx: int) -> Dict:
```

Comment:
Can you put these changes in the other examples PR -- just want to separate out docs from example fixes.